### PR TITLE
#163296321 Fix Update Profile Avatar Field.

### PR DESCRIPTION
**Pull request by verenceLola**
#### What does this PR do?
Fixes issues when updating the avatar url of the user profile.
#### Description of Task to be completed?
 - Fix issue when updating the profile avatar field not being updated on the database.
 - Fix issue to allow the avatar field to pick only valid url links and not any string value.
#### How should this be manually tested?
To test the application follow the steps below.
 - Clone the repo using `git clone https://github.com/andela/ah-technocrats.git`
 - `cd ah-technocrats`
 - `git checkout fix-avatar-update-163296321` to switch to current branch
 - run `virtualenv env -p python3` to create a virtual environment for the application
 - Activate the environment `source env/bin/activate`
 - Install required modules  using `pip install -r requirements.txt`
 - run `python manage.py test` to test

#### What are the relevant pivotal tracker stories?
[#163296321](https://www.pivotaltracker.com/story/show/163296321)

# Checklist:
- [x] My code follows the style guidelines of this project
- [ ] At least 2 people have reviewed my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

**Review by SilasKenneth — COMMENTED** [review]
Looks good @verenceLola, could you fix the CORS issue too? Or it's in a different bug story?

**Comment by verenceLola** [comment]
I have it on a different PR. Go ahead and review it

File: authors/apps/profiles/serializers.py [hunk 1]
```diff
@@ -40,7 +41,4 @@ class Meta:
             'notifications_enabled',
         )
 
-    def get_avatar(self, object):
-        """return the user avatar, or default avatar if non is set"""
-        if object.avatar:
-            return object.avatar
+
```
Comment (dotNesh) [inline authors/apps/profiles/serializers.py]: Leave a line at the end of the file.

**Review by dotNesh — DISMISSED** [review]
LGTM!

File: authors/apps/profiles/serializers.py (hunk 1 above)
Comment (verenceLola) [reply authors/apps/profiles/serializers.py]: I have fixed that

**Review by dotNesh — APPROVED** [review]

**Pull request merged**